collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesActivity.java

```diff
@@ -14,111 +14,56 @@
 
 package org.odk.collect.android.preferences;
 
-import android.accounts.Account;
-import android.accounts.AccountManager;
 import android.content.Intent;
 import android.content.SharedPreferences;
 import android.content.SharedPreferences.Editor;
 import android.net.Uri;
 import android.os.Bundle;
-import android.preference.CheckBoxPreference;
-import android.preference.EditTextPreference;
-import android.preference.ListPreference;
-import android.preference.Preference;
-import android.preference.Preference.OnPreferenceChangeListener;
+import android.preference.PreferenceActivity;
 import android.preference.PreferenceManager;
 import android.preference.PreferenceScreen;
 import android.provider.MediaStore.Images;
-import android.text.InputFilter;
-import android.text.Spanned;
 import android.util.Log;
 
-import com.google.android.gms.analytics.GoogleAnalytics;
-
 import org.javarosa.core.services.IPropertyManager;
 import org.odk.collect.android.R;
 import org.odk.collect.android.logic.FormController;
 import org.odk.collect.android.logic.PropertyManager;
 import org.odk.collect.android.utilities.MediaUtils;
-import static org.odk.collect.android.preferences.PreferenceKeys.*;
 
-import java.util.ArrayList;
-import java.util.List;
+import static org.odk.collect.android.preferences.PreferenceKeys.KEY_SPLASH_PATH;
 
 /**
  * Handles general preferences.
  *
  * @author Thomas Smyth, Sassafras Tech Collective (tom@sassafrastech.com;
  *         constraint behavior option)
  */
-public class PreferencesActivity extends AppPreferenceActivity implements OnPreferenceChangeListener {
-    private static final String t = "PreferencesActivity";
+public class PreferencesActivity extends PreferenceActivity {
     public static final String INTENT_KEY_ADMIN_MODE = "adminMode";
     protected static final int IMAGE_CHOOSER = 0;
-
+    private static final String t = "PreferencesActivity";
```

Comment:
https://medium.com/@elye.project/the-danger-of-using-class-getsimplename-as-tag-for-fragment-5cdf3a35bfe2#.yiz5an5eb

But I do agree that the constant name should be TAG. The codebase is inconsistent so we can just change this when we do a style check across all classes.